test/migrations.unit.spec.js

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'chai';
+
+import fullSchema1010ez from '../src/js/hca/config/form';
+import fullSchema1990 from '../src/js/edu-benefits/1990/config/form';
+import fullSchema1990e from '../src/js/edu-benefits/1990e/config/form';
+import fullSchema1990n from '../src/js/edu-benefits/1990n/config/form';
+import fullSchema1995 from '../src/js/edu-benefits/1995/config/form';
+import fullSchema5490 from '../src/js/edu-benefits/5490/config/form';
+import fullSchema5495 from '../src/js/edu-benefits/5495/config/form';
+import fullSchema527EZ from '../src/js/pensions/config/form';
+import fullSchema530 from '../src/js/burials/config/form';
+import fullSchema10007 from '../src/js/pre-need/config/form';
+
+describe('form migrations:', () => {
+  it('should have a length equal to the version number', () => {
+    const configs = [
+      fullSchema1010ez,
+      fullSchema1990,
+      fullSchema1990e,
+      fullSchema1990n,
+      fullSchema1995,
+      fullSchema5490,
+      fullSchema5495,
+      fullSchema527EZ,
+      fullSchema530,
+      fullSchema10007
+    ];
+    configs.forEach(form => {
+      if (form.migrations || form.version > 0) {
+        expect(form.migrations.length).to.equal(form.version);
```

Comment:
Can we make sure they're SiP-enabled first? Something like:

```js
import { isSIPEnabledForm } from '../src/js/user-profile/helpers';
...
if (isSIPEnabledForm(form.formID) && (form.migrations || form.version > 0)) {
  ...
}
```

Reply:
Thanks @cvalarida. I could imagine this test failure being helpful even if SIP has been disabled for a form (or even in the unlikely event that it has not yet been enabled). Or would it make the most sense to combine the work involved in resolving such a test failure along with SIP re-enablement work?

Reply:
Yeah, after talking with you in slack, I'm with you. If the form has never been SiP-enabled, it won't have migrations or a version > 0, and if it's only temporarily disabled, we still want to make sure the migrations and form version match up. You're 101% right!

Disregard my previous comment. 👍